lib/main/STM32G4/Drivers/STM32G4xx_HAL_Driver/Inc/stm32g4xx_hal_flash.h

```diff
@@ -841,15 +841,15 @@ HAL_StatusTypeDef  FLASH_WaitForLastOperation(uint32_t Timeout);
 #define FLASH_SIZE_DATA_REGISTER        FLASHSIZE_BASE
 
 #if defined (FLASH_OPTR_DBANK)
-#define FLASH_SIZE                      ((((*((uint16_t *)FLASH_SIZE_DATA_REGISTER)) == 0xFFFFU)) ? (0x200UL << 10U) : \
+#define HAL_FLASH_SIZE                      ((((*((uint16_t *)FLASH_SIZE_DATA_REGISTER)) == 0xFFFFU)) ? (0x200UL << 10U) : \
```

Comment:
Is this change actually needed? This could probably be worked around by a `#define HAL_FLASH_SIZE FLASH_SIZE` outside of `lib/`.

Reply:
You mean delete the entire `HAL_FLASH_SIZE` macro and replace it with `#define HAL_FLASH_SIZE FLASH_SIZE outside of lib/`?

Reply:
In that case, it is equivalent to the original `FLASH_SIZE` macro here being deleted and `FLASH_SIZE` from mk replacing it, right?

Reply:
Ah, this rename is done to solve a name clash? This wasn't clear to me. If this is the case then renaming seems to be the best option, and probably add a comment explaining the reason for the rename?

Reply:
👍